### PR TITLE
Hide fundraising actions when needed

### PR DESCRIPTION
**Pull request by JohnOrigan**
* https://fr.boardgamearena.com/bug?id=20524
* Code idea: By default FundRaising actions are hidden (new images added for that), then over decades they become visible
* Note1: My first BGA project, my first web project, so feel free to refactor or totally ignore this request :)
* Note2: Tested in BGA, inside "testjfroux"
* buildings_small_final.png was also changed (pasted here)
*  image

### CODE REVIEW

File: cityofthebigshoulders.js [hunk 1]
```diff
@@ -988,8 +1015,8 @@ function (dojo, declare) {
             // advertising for ad blockers
             this.addTooltip( "banana", "", _( "When you place a Partner here and pay $20, increase your company’s Appeal by 1 space on the Appeal Track. Additionnally, you may become the starting player for the Action Phase. This change in turn order takes effect immediately." ));
             this.addTooltip( "fundraising_40", "", _( "When you place a Partner here, your company gains $40 from the bank." ));
-            this.addTooltip( "fundraising_60", "", _( "When you place a Partner here, your company gains $60 from the bank." ));
-            this.addTooltip( "fundraising_80", "", _( "When you place a Partner here, your company gains $80 from the bank." ));
+            this.addTooltip( "fundraising_60", "", _( "When you place a Partner here, your company gains $60 from the bank. Available from 3rd decade." ));
```
Comment (ggohierroy) [inline cityofthebigshoulders.js]: Available from the 3rd decade only.

File: cityofthebigshoulders.js [hunk 2]
```diff
@@ -988,8 +1015,8 @@ function (dojo, declare) {
...
+            this.addTooltip( "fundraising_80", "", _( "When you place a Partner here, your company gains $80 from the bank. Available at 5th decade only." ));
```
Comment (ggohierroy) [inline cityofthebigshoulders.js]: Available in the 5th decade only.

File: cityofthebigshoulders.js [hunk 3]
```diff
@@ -916,6 +918,31 @@ function (dojo, declare) {
...
+                case 3:
+					dojo.style('fundraising_80', 'background-image', 'none');
+                case 2:
+					dojo.style('fundraising_60', 'background-image', 'none');
+                case 1:
+					dojo.style('fundraising_40', 'background-image', 'none');
+                case 0:
+                    break;
+            }
+        },
```
Comment (ggohierroy) [inline cityofthebigshoulders.js]: This works, so I'll merge it as is.

Personally, I would've preferred doing it without a switch case with pass through cases.

e.g. if(round >= 1 && phase >= 1) then remove background for $40

**Review by ggohierroy — CHANGES_REQUESTED** [review]
Good work, I'll merge this once you've changed the message strings to what I suggested. Thanks!

File: cityofthebigshoulders.js (hunk 3 above)
Comment (JohnOrigan) [reply cityofthebigshoulders.js]: True, much better that way (I fixed too)

**Comment by JohnOrigan** [comment]
Change done (I also fixed tab vs space)

**Review by ggohierroy — APPROVED** [review]
Thanks, merging!

**Pull request merged**